TheFourFifths/consus
# Added check in API endpoint server-side

### Summary

Added API endpoint for item check-in on server
### Checklist

Finally, is it ready to be reviewed and merged: :white_check_mark: 
### How to Review
1. Check if there is anything I missed.
2. Check that the API endpoint is sufficient for checking in an item.
### Links

Put links to Jira tickets, and anything else, here.
- [TFF-73](https://msoese.atlassian.net/browse/TFF-73)
